internal/pkg/pluginmanager/manager.go

```diff
@@ -20,15 +27,15 @@ func DownloadPlugins(conf *configloader.Config) error {
 		pluginFileName := configloader.GetPluginFileName(&tool)
 		if _, err := os.Stat(filepath.Join(pluginsDir, pluginFileName)); errors.Is(err, os.ErrNotExist) {
 			// plugin does not exist
-			log.Printf("=== now downloading plugin: %s, version: %s ===", pluginFileName, tool.Version)
+			log.Printf("=== downloading plugin: %s ===", pluginFileName)
```

Comment:
if the version should be removed or not

Reply:
I deleted it for the version info has contained in the filename.